### PR TITLE
Lesson Plans from Magdalena GH

### PR DESCRIPTION
**Pull request by richarddushime**
@SamGuay 
Hope you are doing well 
As we did , i followed the same methods but instead everything is appearing on this http://localhost:1313/lesson-plans/ page as sections 

when i http://localhost:1313/lesson-plans/ND/ 
its empty 

What do you think might be causing it ?

### CODE REVIEW

**Comment by SamGuay** [comment]
> when i http://localhost:1313/lesson-plans/ND/
> its empty

HUGO _urlify_ directory names, so anything with capital letters gets standardized. IIRC, it should be accessible at http://localhost:1313/lesson-plans/nd/ 

> As we did , i followed the same methods but instead everything is appearing on this http://localhost:1313/lesson-plans/ page as sections 

Seems we overlooked this earlier. Let's deal with this later. This simplest way would probably be to nest `ND` at the parent level (`/content/ND/`) then modify the [url parameter in the front matter of the index.md](https://gohugo.io/content-management/urls/#url) to keep the "lessons-plan" if thats desired.  Else we can always convert the lessons plans page to a section page.. but the first option is easier to implement.

**Comment by richarddushime** [comment]
> > when i http://localhost:1313/lesson-plans/ND/
> > its empty
> 
> HUGO _urlify_ directory names, so anything with capital letters gets standardized. IIRC, it should be accessible at http://localhost:1313/lesson-plans/nd/
> 
> > As we did , i followed the same methods but instead everything is appearing on this http://localhost:1313/lesson-plans/ page as sections
> 
> Seems we overlooked this earlier. Let's deal with this later. This simplest way would probably be to nest `ND` at the parent level (`/content/ND/`) then modify the [url parameter in the front matter of the index.md](https://gohugo.io/content-management/urls/#url) to keep the "lessons-plan" if thats desired. Else we can always convert the lessons plans page to a section page.. but the first option is easier to implement.

As suggested All is working well here : `http://localhost:1313/nd/`
i have just made a commit 
you can check it out , with page editings , broken links etc... 
i will keep improving 
Meanwhile let me ask Flavio and MGH 
what could be the best page slug

**Comment by richarddushime** [comment]
@SamGuay 
Hope you are doing well 
would you look into this PR ? 
I would like to know where the main link URL can be placed 
under what and where

**Comment by SamGuay** [comment]
> I would like to know where the main link URL can be placed under what and where

I'll let @flavioazevedo answer where he'd like it. Not sure if this would be a new menu item or simply a new item in Syllabus or Lessons plan or Neurodiversity? 

Then I'll review it! :)

**Comment by richarddushime** [comment]
Hi @SamGuay @flavioazevedo 
After  looking into the neurodiversity  [page here](https://forrt.org/neurodiversity/)  there is a section that talks about Neurodiversify your curriculum  resources , 
what i did i just included  a link to the full materials on the forrt website instead of  directing to the external website  for Magdalena

**Comment by richarddushime** [comment]
> Hi @SamGuay @flavioazevedo After looking into the neurodiversity [page here](https://forrt.org/neurodiversity/) there is a section that talks about Neurodiversify your curriculum resources , what i did i just included a link to the full materials on the forrt website instead of directing to the external website for Magdalena
> 
> check the last commit

see the Preview here[ Neurodiversify your curriculum](https://preview--forrt-prev.netlify.app/neurodiversity-lessonbank/)

or From the neurodiversity page [ND section below ](https://preview--forrt-prev.netlify.app/neurodiversity/)

File: content/neurodiversity-lessonbank/community_psychology-diversity/index.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+*   Culture, Race and Gender
+
+*   Age, social class and sexual orientation
+
+*   Ability / Disability and Religion & Spirituality
+
+*   Privilege and intersectionality
+
+TEXT
```
Comment (SamGuay) [inline content/neurodiversity-lessonbank/community_psychology-diversity/index.md:85]: What does this refer to?

File: content/neurodiversity-lessonbank/community_psychology-diversity/index.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+(to be be done before or after the lesson)
+
+Jason, Leonard A.; Glantsman, Olya; O'Brien, Jack F.; and Ramian, Kaitlyn N., "Introduction to Community Psychology: Becoming an Agent of Change" (2019). College of Science and Health Full Text Publications. 1. https://via.library.depaul.edu/cshtextbooks/1
+
+Flavio Azevedo, Sara Middleton, Jenny Mai Phan, Steven Kapp, Amélie Gourdon-Kanhukamwe, Bethan Iley, Mahmoud Elsherif, & John J. Shaw. Navigating Academia as Neurodivergent Researchers: Promoting Neurodiversity Within Open Scholarship. APS Observer October 2022
+
+Elsherif, M. M., Middleton, S. L., Phan, J. M., Azevedo, F., Iley, B. J., Grose-Hodge, M., … Dokovova, M. (2022, June 20). Bridging Neurodiversity and Open Scholarship: How Shared Values Can Guide Best Practices for Research Integrity, Social Justice, and Principled Education.
+
+Footer © Magdalena Grose-Hodge for FORRT (Framework of Open and Reproducible Research Training) About ND/Community Psychology - Diversity at main · MSGrose-Hodge/ND
```
Comment (SamGuay) [inline content/neurodiversity-lessonbank/community_psychology-diversity/index.md:112]: I think that footer line can be removed from all the pages, it is irrelevant as author is acknowledged throughout the pages

File: content/neurodiversity/neurodiversity.md [hunk 3]
```diff
@@ -235,13 +235,13 @@ We recommend this talk for educators and researchers who wish to integrate inter
 
 {{< /spoiler >}}
 
-### Neurodiversify Your Curriculum
+### [ Neurodiversify Your Curriculum](../neurodiversity-lessonbank)
```
Comment (SamGuay) [inline content/neurodiversity/neurodiversity.md:238]:
```suggestion
### [Neurodiversify Your Curriculum](../neurodiversity-lessonbank)
```

**Comment by richarddushime** [comment]
Thank you, i m addressing the changes  right now

**Comment by flavioazevedo** [comment]
I know this will sound not great, but I will approve the changes, and see how it looks on the website (I tried by best to understand how I try to pull this to my local but couldn't).

**Review by flavioazevedo — APPROVED** [review]
Looks great, thank you Richard.

**Comment by richarddushime** [comment]
> I know this will sound not great, but I will approve the changes, and see how it looks on the website (I tried by best to understand how I try to pull this to my local but couldn't).

I understand 
Since it is merged, I will address  the changes  on a new branch 
Let me ask Magdalena  for a review since she is the author

**Comment by flavioazevedo** [comment]
I think Magdalena will help make it look the way she wants (format, figures, etc.), and I would then like to talk to you about how to structure the page of ND (to highlight that resource) and also link it back to the lesson plans initiative (i.e., include the links to the lesson plans there) as well.

**Comment by richarddushime** [comment]
> I think Magdalena will help make it look the way she wants (format, figures, etc.), and I would then like to talk to you about how to structure the page of ND (to highlight that resource) and also link it back to the lesson plans initiative (i.e., include the links to the lesson plans there) as well.

Great, I'm available

**Pull request merged**